### PR TITLE
app-admin/pprocm: update HOMEPAGE, SRC_URI

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This updates the `HOMEPAGE` of `app-admin/pprocm`. The original seem to have changed owner.
Luckily the sources from the owner are on github too, however, unfortunately without a release.

I've added a `r4` which uses the sources from github. This one has unstable keywords, but if it's fine i can add it directly to stable.

### CODE REVIEW

File: app-admin/pprocm/pprocm-1.0-r4.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+MY_PN="PProcM"
+MY_P="${MY_PN}-${PV}"
+
+GIT_COMMIT="98ca63f43afb9ec8491cceb81f62850ff00379e0"
+
+DESCRIPTION="ncurses-based program to monitor CPU, disk, network and memory usage"
+HOMEPAGE="https://github.com/ZachGoldberg/Perl-Proc-Monitor-PProcM/"
+SRC_URI="https://github.com/ZachGoldberg/Perl-Proc-Monitor-PProcM/archive/${GIT_COMMIT}.tar.gz
+ -> ${P}.tar.gz"
```
Comment (ulm) [inline app-admin/pprocm/pprocm-1.0-r4.ebuild:14]:
```
$ pkgcheck scan --net        
app-admin/pprocm
  WhitespaceFound: version 1.0-r4: ebuild has leading whitespace on on line: 14
```

Comment (mm1ke) [reply app-admin/pprocm/pprocm-1.0-r4.ebuild]: thanks, fixed

File: app-admin/pprocm/pprocm-1.0-r4.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="PProcM"
+MY_P="${MY_PN}-${PV}"
```
Comment (ulm) [inline app-admin/pprocm/pprocm-1.0-r4.ebuild:7]: Looks like `MY_P` isn't used. (Sorry, missed this last time.)

Comment (mm1ke) [reply app-admin/pprocm/pprocm-1.0-r4.ebuild]: nice catch, fixed it. Also starting with a capital letter in the `DESCRIPTION` now.

**Comment by ulm** [comment]
Thanks.

**Pull request closed without merging**